Dockerfile

```diff
@@ -55,3 +55,9 @@ COPY . /opt/app-root/src
 
 RUN chgrp -R 0 /opt/app-root && \
     chmod -R g=u /opt/app-root
+
+# TODO remove the following block before merge!
```

Comment:
Open problem: elinks dependency

@Elkasitu You mentioned you are working on it?

Reply:
Done